.github/workflows/test.yml

```
@@ -38,7 +39,7 @@ jobs:
             -e JEKYLL_GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} \
             portal build \
             --env=production \
-            --verify \
+            --verify
```

Without escaping the newline character, all arguments following `--verify` will be ignored.

```suggestion
            --verify \
```